specification.md

```diff
@@ -0,0 +1,148 @@
+# Specification
+
+The following values are part of Security Insights Specification `v1.0.0`
+
+## Header
+Here is an example of the header section, which holds high-level information about the project and its security documentation.
+
+```yaml
+header:
+  schema-version: 1.0.0
+  last-updated: '2021-09-01'
+  last-reviewed: '2022-09-01'
+  expiration-date: '2023-08-31T10:10:09.000Z'
+  project-url: https://github.com/foo/bar
+  commit-hash: 4dbf78ebc006ee5f668c0a74876ef8d6db9485be
+  project-release: '1.2.0'
+  changelog: https://github.com/foo/changelog.md
+  license: https://git.foo/license
+  parent-security-yaml: https://blah.com/ossf-security.yaml
+```
+
+Now let’s break that down:
+- `schema-version` (Required): Provide the version of the specification that you are adhering to.
+  - **String**
+- `last-updated` (Required): Date of the most recent change to this file.
+  - **Date**
+- `last-reviewed` (Required): Date this file was last reviewed to confirm holistic accuracy.
+  - **Date**
+- `expiration-date` (Required): Date this file should no longer be considered valid. Maximum one year from the date it was last reviewed.
+  - **Date-time**
+- `project-url` (Required): URI for the project repository that this document pertains to.
+  - **String** beginning with **https://**
+- `commit-hash` (Required): The last commit on the repository that this document pertains to.
+  - **String** matching the git SHA format
+- `changelog`: URI for the changelog for the project.
+  - **String** beginning with **https://**
+- `license`: URI for the license for the corresponding version of the project.
+  - **String** beginning with **https://**
+- `parent-security-yaml`: If the document is mirrored from another, give a URI pointing to the original.
+  - **String** beginning with **https://**
+
+## Project Lifecycle
+The project-lifecycle section contains information about how the project is maintained. This is the first of many places where we’ll see conditional requirements in the schema.
```

Comment:
Proposed:
The purpose of the `project-lifecycle` section is to describe support, maintenance, and fitness expectations for the consumer. For example, a project may be just getting started, be mature, be in "bugfix only" mode, or be completely abandoned. It helps clarify expectations all around.

Reply:
changed